### PR TITLE
fix(samples): Update execution_order, loop_output samples to v2 pipelines

### PR DESCRIPTION
**Pull request by junggil**
Update execution_order, loop_output samples to v2 pipelines

**Checklist:**
- [v] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: samples/core/loop_output/loop_output.py [hunk 1]
```diff
@@ -12,22 +12,30 @@
...
+import os
+from kfp import dsl
 
+# In tests, we install a KFP package from the PR under test. Users should not
+# normally need to specify `kfp_package_path` in their component definitions.
+_KFP_PACKAGE_PATH = os.getenv('KFP_PACKAGE_PATH')
 
-@components.create_component_from_func
+
+@dsl.component(kfp_package_path=_KFP_PACKAGE_PATH)
```
Comment (connor-mccarthy) [inline samples/core/loop_output/loop_output.py:23]: Consider omitting use of `kfp_package_path` since this is a sample not a test.

**Review by connor-mccarthy — COMMENTED** [review]
/lgtm

File: samples/core/loop_output/loop_output.py (hunk 1 above)
Comment (junggil) [reply samples/core/loop_output/loop_output.py]: Thanks for the comment, just addressed.

**Review by connor-mccarthy — APPROVED** [review]

**Pull request merged**